src/components/signup/SignUpForm.tsx

```diff
@@ -26,9 +27,20 @@ const SignUpForm: React.FC<SignUpFormProps> = ({
       className="bg-white rounded-2xl flex flex-col items-center w-[95%] md:w-[600px] mx-auto min-h-screen pb-8 shadow-xl"
     >
       <div className="w-full flex flex-col items-center py-8 shadow-lg rounded-b-lg">
-        <h1 className="font-bold text-2xl md:text-3xl text-secondary mb-8">
-          {title}
-        </h1>
+        <div className="text-center flex items-center justify-center mb-8 relative w-full">
+          {step === 1 || (
```

Comment:
what are you trying to achieve with
```ts
{step === 1 || ()}
```
@joshxfi

Reply:
The back button will not show on step 1 @princejoogie

Reply:
please replace it with
```ts
{step !== 1 && ()}
```

Reply:
this applies for the whole project, never use `||` to conditionally render something. either use ternary or a condition with `&&`

Reply:
aighty got it